src/main/resources/application-custom.yaml

```diff
@@ -1,91 +1,119 @@
+#Adds the option to go to eg. http://localhost:8080/actuator/health for seeing the running configuration
+#see https://docs.spring.io/spring-boot/docs/current/reference/html/actuator.html#actuator.endpoints
+management:
+  endpoints:
+    web:
+      exposure:
+        include: "health,prometheus"
 spring:
+  main:
+    allow-circular-references: true
+    #allow-bean-definition-overriding: true
+  batch.job.enabled: false
+  flyway:
+    enabled: false
+    check-location: false
+    baselineOnMigrate: true
   datasource:
     url: ${datasource.url}
     username: ${datasource.username}
     password: ${datasource.password}
     driverClassName: ${datasource.driver}
     max-active: 15
+
+    # database connection pool size
+    hikari:
+      maximum-pool-size: 10
   jpa:
     properties:
-      hibernate.dialect: ${hibernate.dialect}
       hibernate.format_sql: false
       hibernate.show_sql: false
+      hibernate.dialect: ${hibernate.dialect}
       hibernate.hbm2ddl.auto: update
       hibernate.jdbc.batch_size: 20
       hibernate.cache.use_query_cache: false
       hibernate.cache.use_second_level_cache: false
       hibernate.cache.use_structured_entries: false
       hibernate.cache.use_minimal_puts: false
-###    These settings will enable fulltext search with lucene
-      hibernate.search.enabled: true
+  ###    These settings will enable fulltext search with lucene or elastic
+      hibernate.search.enabled: false
+  ### lucene parameters
       hibernate.search.backend.type: lucene
```

Comment:
Since we have search disabled, I think we should comment-out all of `hibernate.search.*` settings on lines 41-45.